conda/recipes/dask-cuda/recipe.yaml

```
@@ -46,6 +46,7 @@ requirements:
     - pandas >=1.3
     - rapids-dask-dependency =${{ minor_version }}
     - zict >=2.0.0
+    - __linux
```

Since we are setting a GLIBC minimum in wheels to match what we build and test against, we can do the same thing in Conda like so

```suggestion
    - __linux
    - __glibc >=2.28
```